datepicker/src/main/scala/lucuma/react/datepicker/package.scala

```diff
@@ -24,51 +21,31 @@ package object datepicker {
       )
   }
 
-  object LocalDateBuilder {
-    def fromJsDate(jsDate: js.Date): LocalDate =
+  extension (jsDate: js.Date) {
+    def fromJsDate: LocalDate =
       LocalDate.of(
         jsDate.getFullYear().toInt,
         jsDate.getMonth().toInt + 1,
         jsDate.getDate().toInt
       )
   }
 
-  implicit class JSUndefOrNullOrTuple2Ops[A](
-    val value: js.UndefOr[DateOrRange]
-  ) extends AnyVal {
-    def toEitherOpt: Option[Either[(A, A), A]] =
-      value.toOption
-        .flatMap(valueOrNull => Option(valueOrNull.asInstanceOf[A | js.Tuple2[A, A]]))
-        .map { valueOrTuple =>
-          if (js.Array.isArray(valueOrTuple))
-            Left(valueOrTuple.asInstanceOf[js.Tuple2[A, A]])
-          else
-            Right(valueOrTuple.asInstanceOf[A])
-        }
-
-    def toOpt: Option[A] =
-      toEitherOpt.flatMap(_.toOption)
-
-    def toTupleOpt: Option[(A, A)] =
-      toEitherOpt.flatMap(_.left.toOption)
-
-    def toLocalDateEitherOpt(implicit
-      ev: A <:< js.Date
-    ): Option[Either[(LocalDate, LocalDate), LocalDate]] =
-      toEitherOpt.map {
-        case Left((d1, d2)) =>
-          Left((LocalDateBuilder.fromJsDate(ev(d1)), LocalDateBuilder.fromJsDate(ev(d2))))
-        case Right(d)       =>
-          Right(LocalDateBuilder.fromJsDate(ev(d)))
-      }
-
-    def toLocalDateOpt(implicit ev: A <:< js.Date): Option[LocalDate] =
-      toLocalDateEitherOpt.flatMap(_.toOption)
-
-    def toLocalDateTupleOpt(implicit ev: A <:< js.Date): Option[(LocalDate, LocalDate)] =
-      toLocalDateEitherOpt.flatMap(_.left.toOption)
-  }
-
-  implicit def builder2VdomElement(builder: StBuildingComponent[?]): VdomElement =
-    builder.build
-}
+// DatePicker only works in local timezone, so we trick it by adding the timezone offset.
```

Comment:
Have you considered exposing `LocalDateTime` instead of `Instant`? Since the picker doesn't specify a timezone that may be more accurate.

Reply:
I used Instant because that was the code I moved from Explore.

Reply:
I seem to recall a lot of time was spent getting the times to come out right. I didn't really want to mess with that.